src/app/admin/clients/page.tsx

```
@@ -363,69 +373,42 @@ export default function AdminClientsPage() {
-                    // Create and trigger download
-                    const blob = new Blob([csvContent], { type: 'text/csv;charset=utf-8;' })
-                    const link = document.createElement('a')
-                    link.href = URL.createObjectURL(blob)
-                    link.download = `clients_export_${new Date().toISOString().split('T')[0]}.csv`
-                    link.click()
-                    URL.revokeObjectURL(link.href)
-
+                    // Generate CSV content for selected clients
+                    const selectedClientsData = clients.filter(client =>
+                        selectedClients.includes(client.id)
+                    )
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)